### PR TITLE
Improve Landing page

### LINKED ISSUE #332: Improve landing page design

### PR DESCRIPTION
**Pull request by zognin**
Closes #332 

I didn't extract the components
Also I'm not sure about the stats section

@ reviewer: can you see if the content and look is okay first?

### CODE REVIEW

File: frontend/styles/indexStyles.ts [hunk 1]
```diff
@@ -1,197 +1,313 @@
...
+  backgroundImage: `url(${MobileHeaderImage.src})`,
+  ...headlineContainerSx,
 };
 
-export const secondaryTranslucentOverlaySx: SxProps = {
-  textAlign: 'center',
-  color: '#FFFFFF',
-  backgroundColor: theme.palette.secondaryTranslucent.main,
+export const instructionsContainerSx: SxProps = {
+  backgroundColor: alpha('#F1EAE6', 0.2),
```
Comment (sivayogasubramanian) [inline frontend/styles/indexStyles.ts:51]: Don't really like this as it adds an ugly horizontal line

**Review by sivayogasubramanian — COMMENTED** [review]
Some suggestions/comments

image

Instead of the picture above, can we cycle through the pictures that we took?

<hr>

After the section above, we should have a how it works section that is like a flow chart that tells the reader how giving coupons works.

image

The info here should be in the flow chart i think.

<hr>

image
image

After we add the how it works section, a lot of the things here can be removed and I think we should restrict it to max 2 sentences as no one will go read so much info

<hr>

image

The color is way too distracting.

<hr>

image

Feels like a repetition of the things above in the instruction section

<hr>

image

Can we put this inside a container so it looks something like this?

image

**Comment by zognin** [comment]
The last part is meant to repeat to get users to do the call to action stuff, otherwise they may just stop there and do nothing.

File: frontend/pages/index.tsx [hunk 2]
```diff
@@ -1,247 +1,425 @@
...
-        <Grid sx={statsSectionGridSx} container spacing={2}>
-          <StatItem imgSrc="/icon-charity.png" stat="$1,600" subtitle="Additional funds raised for charity" />
+              <Typography sx={descriptionTextSx}>Experience and share the joy of giving by</Typography>
 
-          <StatItem imgSrc="/icon-voucher.png" stat="160" subtitle="Coupons issued" />
+              <Typed
+                style={descriptionTextSx}
+                strings={[
+                  'empowering others to give',
+                  'learning how your donations helps charities',
```
Comment (bharathcs) [inline frontend/pages/index.tsx:220]:
```suggestion
                  'learning how your donations help charities',
```

**Comment by zognin** [comment]
I added horizontal lines to make the sectioning more obvious. But I removed/changed the background color.

**Review by sivayogasubramanian — APPROVED** [review]
LGTM!

**Pull request merged**